### PR TITLE
fix: qmljs hightlight

### PR DESCRIPTION
**Pull request by Decodetalkers**
Since ecma is still not fixed, should we add the part of highlight ourself?

### CODE REVIEW

File: queries/qmljs/highlights.scm [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ; temporarily removed the ecma inherit due to it breaking
 "pragma" @keyword.import
 
+(ui_import
+  "import" @keyword.import)
```
Comment (lucario387) [inline queries/qmljs/highlights.scm:5]: Can't this just be `"import" @keyword.import`

Comment (Decodetalkers) [reply queries/qmljs/highlights.scm:5]: Emm. I just worry that maybe it will find other place..

File: queries/qmljs/highlights.scm [hunk 2]
```diff
@@ -1,6 +1,12 @@
 ; temporarily removed the ecma inherit due to it breaking
 "pragma" @keyword.import
 
+"import" @keyword.import
```
Comment (lucario387) [inline queries/qmljs/highlights.scm:4]: Grouping this, possibly

Comment (Decodetalkers) [reply queries/qmljs/highlights.scm]: done

**Review by ObserverOfTime — COMMENTED** [review]
If whatever issue blocks inheritance cannot be fixed then *all* relevant parent highlights should be added.

**Pull request closed without merging**